src/toil/provisioners/abstractProvisioner.py

```diff
@@ -52,7 +46,7 @@ class AbstractProvisioner(object):
     __metaclass__ = ABCMeta
 
 
-    def __init__(self, config=None, batchSystem=None):
+    def __init__(self, config=None):
         """
```

Comment:
The batch system was only referenced in the higher level logic that I moved into the cluster scaler class so there was no need to keep it in this class as well. But yeah the doc string should be updated